htdocs/adherents/list.php

```diff
@@ -83,7 +83,6 @@
 
 // List of fields to search into when doing a "search in all"
 $fieldstosearchall = array(
-    'd.rowid'=>'Ref',
```

Comment:
I would prefer a test on 
if ($db->type == 'pgsql') to exclue the field only on posgresql
because search on ref is the common usage.

Reply:
Agreed, I'll look into it.